.travis.yml

```diff
@@ -12,21 +12,24 @@ services:
 before_install:
   - sudo /etc/init.d/mysql stop
   - sudo apt-get update
```

Comment:
This seems obsolete as well. An update without an upgrade or install doesn't make sense to me.